src/cockpit/polkit.py

```diff
@@ -0,0 +1,154 @@
+import asyncio
```

Comment:
Copyright header is missing.  Sorry for not catching that earlier.

Reply:
Right! Added.